xbmc/addons/kodi-addon-dev-kit/include/kodi/AddonBase.h

```diff
@@ -82,7 +82,9 @@ typedef enum ADDON_STATUS
   ///
   ADDON_STATUS_UNKNOWN,
 
+  /// @warning obsolete and no more used from Kodi!
   ///
+  /// Use kodi::SetSetting... during end to store changed settings
   ADDON_STATUS_NEED_SAVEDSETTINGS,
```

Comment:
If it is no longer used, remove it, please. Yes I know that some addons need to be bumped for this, but still.